com.github.d4nj1.tlpui.yml

```diff
@@ -0,0 +1,34 @@
+app-id: com.github.d4nj1.tlpui
+command: tlpui
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+finish-args:
+  - --socket=fallback-x11
```

Comment:
you need --share=ipc as well if you're going to allow x11.

also - fallback-x11 is designed for use in a scenario where --socket=wayland is also used. Does this not support wayland?

Reply:
Hi,
I created this config more than a year ago and only adopted to yaml notation. I guess some stuff has changed since then. I do not even find the option for x11-fallback in the docs anymore. https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html

What do I have to add if would like to support both X11 and Wayland? Will it be auto discovered if I avoid adding `--socket=wayland` and `--socket=x11`?

Reply:
Sorry to bother, it's actually documented here: https://docs.flatpak.org/en/latest/sandbox-permissions.html